src/libostree/ostree-repo-commit.c

```diff
@@ -1649,13 +1651,7 @@ ostree_repo_prepare_transaction (OstreeRepo     *self,
       guint64 bytes_required = bfree * self->txn.blocksize;
       self->cleanup_stagedir = TRUE;
       g_mutex_unlock (&self->txn_lock);
-      g_autofree char *formatted_free = g_format_size (bytes_required);
-      if (self->min_free_space_percent > 0)
-        return glnx_throw (error, "min-free-space-percent '%u%%' would be exceeded, %s available",
-                           self->min_free_space_percent, formatted_free);
-      else
-        return glnx_throw (error, "min-free-space-size %" G_GUINT64_FORMAT "MB would be exceeded, %s available",
-                           self->min_free_space_mb, formatted_free);
+      return throw_min_free_space_error (self, bytes_required, error);
```

Comment:
Or
>So, we want to report 3rd hunk as reserved_blocks * self->txn.blocksize maybe ?
(because there is requested in every error reporting)

As per my comment in the last PR review.

Reply:
Isn't that just `min-free-space-size` in bytes? (Ignoring the `-percent` case for now). We're dividing by the block size to get min free space in blocks, then re-multiplying it back.

Reply:
Yeah, we don't have direct access to value in bytes as it's abstracted in `min_free_space_calculate_reserved_blocks`